spec/controllers/users_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe ArticlesController, type: :controller do
+    describe '#new' do
+        it 'responds successfully' do
```

Comment:
This seems redundant since there is a test in the `/spec/request` that tests the same.